bin/profiler

```diff
@@ -140,6 +144,22 @@ def plotProfile(hm, outFileName,
 
                 exit(1)
 
+    #If we plot a heatmap, we need to normalize the values to be between 0 and 1
+    if plotType == 'heatmap':
+        maxVal = None
+        for plot in range(numplots):
+            col = plot % plots_per_row
+            row = int(plot / plots_per_row)
+            for data_idx in range(numlines):
+                if perGroup:
+                    row, col = plot, data_idx
+                else:
+                    row, col = data_idx, plot
+                sub_matrix = hm.matrix.get_matrix(row, col)
+                curMax = np.amax(np.__getattribute__(averagetype)(sub_matrix['matrix'], axis=0))
```

Comment:
I like that way of calling np.sum, np.mean, np.median etc. with one line. Do you know a better way to do it?